stem/geometry.py

```diff
@@ -617,7 +617,7 @@ def calculate_centre_of_mass_surface(self, surface_id: int) -> npty.NDArray[np.f
             centre_of_mass += weight * line_centroid
             circumference += weight
 
-        centre_of_mass = centre_of_mass / circumference
+        centre_of_mass /= circumference
```

Comment:
im always scared doing this inplace operation with numpy arrays, as that 1 time it gave wrong results